src/tsconfig.json

```diff
@@ -121,6 +122,7 @@
         "rules/oneLineRule.ts",
         "rules/oneVariablePerDeclarationRule.ts",
         "rules/onlyArrowFunctionsRule.ts",
+        "rules/orderedImportsRule.ts",
```

Comment:
This was added when I ran the `grunt` task.  Should I just not check in this file and ignore it manually?